api/client/proto/authservice.proto

```diff
@@ -1951,6 +1951,9 @@ service AuthService {
     // WatchEvents returns a new stream of cluster events
     rpc WatchEvents(Watch) returns (stream Event);
 
+    // GetEntitledNode returns a node by name only if user is entitled to it with login name.
+    rpc GetEntitledNode(types.EntitlementLookupRequest) returns (types.ServerV2);
```

Comment:
Why we are adding new method when the existing checker can filter the node using the login.